stdlib/sys.ml.in

```diff
@@ -14,10 +15,6 @@
 (*                                                                        *)
 (**************************************************************************)
 
-(* WARNING: sys.ml is generated from sys.mlp.  DO NOT EDIT sys.ml or
-   your changes will be lost.
```

Comment:
If this change goes through, it would be nice if there still was a similar warning (mentioning `sys.ml.in` and `configure`).